docs/the-new-architecture/pure-cxx-modules.md

```diff
@@ -0,0 +1,472 @@
+# Pure C++ Turbo Native Modules
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+Writing a module in C++ is the best way to share platform-agnostic code between Android and iOS. With pure C++ modules, you can write your logic only once and reuse it right away from all the platform, without the need of writing platform specific code.
```

Comment:
This is not advanced to me. This is a base case of a user that needs to write a Native Module and wants share code between platforms. I'm afraid that if we put the "advanced" tag here, it will actually discourage people to try them out.

The other guide has `Advanced` in the title, instead.